README.rst

```diff
@@ -370,7 +370,7 @@ All pull request should pass the test suite, which can be launched simply with:
 
 .. code-block:: sh
 
-    $ make test
+    $ make testall
```

Comment:
What is the difference between `make test` and `make testall`?

I think `make test` is a more natural default, so why not make it use the contents of `testall`, and then rename the current contents of the `test` target to be something more specific/non-default...?

Reply:
`make testall` runs the test matrix with `tox`, as well as linters, documentation building, testing in example projects. `make test` just runs python tests in the current environment.

I think `make test` is good to get quick feedback on a change, since it only runs the test suite. `make testall` is slower, but provides a more complete feedback to contributors. Recommending `make testall` before submitting a PR helps spotting as many mistakes as possible.

Regarding using `make test` for tox, I’m slightly in favor of keeping the current `make test` for the quick feedback and `make testall` for the complete report. Mostly because `make testall` is a good name for running the complete test matrix and lint, docs, examples, and `make test` is less typing to just get a quick feedback on a current change. I can easily be convinced otherwise though.

Reply:
Agree completely... did not realize `make test` runs the full suite, just not across all versions.